tools/cli/test/e2e/cli/merge_test.go

```diff
@@ -122,6 +125,9 @@ func TestMerge(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 		stringResponse := string(resp)
 		require.Error(t, err, stringResponse)
-		assert.Contains(t, stringResponse, "Error: there was a conflict on a Schema component: \"ApiError\"")
+		assert.Contains(t, stringResponse,
```

Comment:
can we display the diff between the two schemas?

Reply:
I think we'd risk printing an enormous string if we do so since the struct used to get the diff is SchemaDiff which has +20 fields + sub-fields

Reply:
we could output in a file maybe then and leave it in the files if the person wants to get details, but no strong opinion feel free to ignore